package/gluon-check-connection/luasrc/usr/bin/gluon-check-connection

```diff
@@ -4,17 +4,11 @@ local unistd = require 'posix.unistd'
 local util = require 'gluon.util'
 local site = require 'gluon.site'
 
-local offline_flag_file = "/tmp/gluon_offline"
+local offline_flag_file = "/var/gluon/check-connection/offline"
```

Comment:
With the changed path, `/var/gluon/check-connection` must be created before the file can be written.

Reply:
I check for the path now and create it now before